modules/products/bitbucket/pre_install.tf

```diff
@@ -1,4 +1,7 @@
 resource "kubernetes_job" "pre_install" {
+  lifecycle {
+    ignore_changes = all
+  }
```

Comment:
`<>` symbols get into job command as:

```
'\u003cbaseUrl\u003ehttp://aabd8af13d1fc4d349258ef27546010c-1160533678.us-east-1.elb.amazonaws.com/confluence\u003c/baseUrl\u003e')
```
As a result, when running terraform apply for the second time, Terraform wants to delete the job and recreate it because pod spec is immutable and cannot be updated once the pod has been created.

Ignoring changes fixes the cycle error. Also, we don't expect to run this job twice i.e. restore environment from a different snapshot if it has been already restored from a snapshot.

Reply:
It is an acceptable trade off as it is less likely to restore multiple dataset snapshots on one environment.